### PR TITLE
add scene scraper for ladyboygold.com and tsraw.com

### PR DESCRIPTION
**Pull request by nrg101**
Add scene scraper that works on tour pages, e.g. https://ladyboygold.com/tour/show_video.php?galid=6635&section=1552&nats=MC4wLjYuNi4wLjAuMC4wLjA

### CODE REVIEW

File: scrapers/LadyboyGold.yml [hunk 1]
```diff
@@ -4,6 +4,12 @@ performerByURL:
     url:
       - ladyboygold.com/index.php
     scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - ladyboygold.com/tour
```
Comment (bnkai) [inline scrapers/LadyboyGold.yml:10]:
``` yaml
     - ladyboygold.com/tour/show_video.php`
```
is probably better to indicate the kind of urls we expect?

File: scrapers/LadyboyGold.yml [hunk 2]
```diff
@@ -4,6 +4,12 @@ performerByURL:
...
+      - tsraw.com
```
Comment (bnkai) [inline scrapers/LadyboyGold.yml:11]: Wasnt able to test this.All links seem to be paywalled?

File: scrapers/LadyboyGold.yml (hunk 1 above)
Comment (nrg101) [reply scrapers/LadyboyGold.yml:10]: i wasn't sure if `/tour/show_video.php` was the only URI path or not, so just left it as `/tour` to be more open-ended... I think it's fine as-is for the purpose of `sceneByURL` though?

**Comment by DogmaDragon** [comment]
Tested with several URLs from <https://www.tsraw.com/index.php?section=1610> and <https://ladyboygold.com/tour/index.php?section=1555>, everything seems okay.

**Pull request merged**